airbyte-webapp-e2e-tests/cypress/pages/replicationPage.ts

```diff
@@ -9,6 +9,9 @@ const destinationNamespaceSource = "div[data-testid='namespaceDefinition-source'
 const destinationNamespaceCustomInput = "input[data-testid='input']";
 const syncModeDropdown = "div[data-testid='syncSettingsDropdown'] input";
 const cursorFieldDropdown = "button[class^='PathPopoutButton_button']";
+const streamFieldNames = "[class^='TreeRowWrapper_rowWrapper'] span";
+const streamDataTypes = "[class^='TreeRowWrapper_rowWrapper'] div:nth-child(2)";
```

Comment:
These selectors work for now, but ideally, we should try to use testIds instead because the implementation can always change and break this. For this PR, it's OK to leave because we also have the problem in other areas.